tests/plugins/toolbar/manual/aria-haspopup.html

```diff
@@ -0,0 +1,66 @@
+<style>
+	table, th, td {
+		border: 1px solid black;
+		border-collapse: collapse;
+	}
+	table {
+		margin: 10px;
+	}
+	td {
+		padding: 3px;
+	}
+</style>
+<div id="editor">
+</div>
+<table>
+	<thead>
+	<tr>
+		<th>Button name:</th>
+		<th>Value of <br> aria-hashpopup:</th>
+	</tr>
+	</thead>
+	<tbody class="output"></tbody>
+</table>
+
+<script>
+	if ( CKEDITOR.env.mobile ) {
```

Comment:
After changing test into such form I don't see need to ignore mobile devices – test should be working just fine on mobiles.